app/pb_kits/playbook/pb_selectable_list/_selectable_list.jsx

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+import React from 'react'
+import classnames from 'classnames'
+import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
+import { globalProps } from '../utilities/globalProps.js'
+import { List } from  '..'
+
+type SelectableListProps = {
+  aria?: object,
+  children?: Node,
+  className?: string,
+  data?: object,
+  id?: string,
+  variant?: 'checkbox' | 'radio',
+}
+
+const SelectableList = (props: SelectableListProps) => {
+  const {
+    aria = {},
+    children,
+    className,
+    data = {},
+    id,
+  } = props
+
+  const ariaProps = buildAriaProps(aria)
+  const classes = classnames(buildCss('pb_selectable_list_kit'), className, globalProps(props))
```

Comment:
className needs to come last so that css modules work correctly in Nitro
```suggestion
  const classes = classnames(buildCss('pb_selectable_list_kit'), globalProps(props), className)
```

Reply:
Fixed it